Exercises/chapter-11/e1-quiet-times.js

```diff
@@ -0,0 +1,23 @@
+async function activityTable(day) {
+    //Get an array of log file names for the specified day
+    const logFileList = await textFile("camera_logs.txt");
+    const logFileArr = logFileList.split("\n")
+        .filter(fn => new Date(fn.match(/\d{4}-\d\d-\d\d/)[0]).getDay() === day);
```

Comment:
Is there a simpler, clearer way of writing this filter? why do you need the regex? Is a one liner actually better than a block?

Reply:
Also, I think fn is supposed to be file name, but fn's standard meaning is function. Being more verbose is usually better. There's no good reason not to just call it fileName here.